client/client.go

```diff
@@ -18,15 +18,19 @@ package client
 import (
 	"context"
 	"crypto/sha256"
+	"time"
 
 	"github.com/google/trillian"
+	"github.com/jpillora/backoff"
```

Comment:
Looks like he has an MIT license in the README.  Opened a ticket for him to create a LICENSE file in his repo.